# HG changeset patch
# User John Donoghue <john.donoghue@ieee.org>
# Date 1396661295 14400
#      Fri Apr 04 21:28:15 2014 -0400
# Branch stable
# Node ID 6535cb2b8e231b1e157eafff166c8fb22cab4b4d
# Parent  3283ce934648e4b3fd550272c87a44b15c6b18de
libqterminal: Initialize m_cursorBlinking value (Bug #41814)

* libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp
  (QConsolePrivate::QConsolePrivat): iInitialize m_cursorBlinking to false.

diff --git a/libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp b/libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp
--- a/libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp
+++ b/libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp
@@ -262,19 +262,19 @@ static void maybeSwapPoints (QPoint& beg
   if (end.y () < begin.y ()
       || (end.y () == begin.y () && end.x () < begin.x ()))
     qSwap (begin, end);
 }
 
 //////////////////////////////////////////////////////////////////////////////
 
 QConsolePrivate::QConsolePrivate (QWinTerminalImpl* parent, const QString& cmd)
-  : q (parent), m_command (cmd), m_hasBlinkingCursor (true),
-    m_cursorType (BlockCursor), m_beginSelection (0, 0),
-    m_endSelection (0, 0), m_settingSelection (false),
+  : q (parent), m_command (cmd), m_cursorBlinking (false),
+    m_hasBlinkingCursor (true), m_cursorType (BlockCursor), 
+    m_beginSelection (0, 0), m_endSelection (0, 0), m_settingSelection (false),
     m_process (NULL), m_inWheelEvent (false)
 {
   log (NULL);
 
   // Possibly detach from any existing console
   log ("Detaching from existing console (if any)...\n");
   FreeConsole ();
   log ("Closing standard IO...\n");
